.changeset/silent-mice-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': minor
```

Comment:
This just introduces a new component to the default app right? i'm thinking it can be `patch` but I'm not 100% sure

Reply:
Done!